packages/trendmicro/data_stream/deep_security/sample_event.json

```
@@ -0,0 +1,67 @@
+{
+    "@timestamp": "2020-09-21T07:21:11.000Z",
+    "agent": {
+        "ephemeral_id": "8070dfb5-8f7f-4717-8bda-e8dd8833bf9c",
+        "id": "eb140869-9022-4dec-b977-6e198e7219b2",
+        "name": "docker-fleet-agent",
+        "type": "filebeat",
+        "version": "8.4.0"
+    },
+    "data_stream": {
+        "dataset": "trendmicro.deep_security",
+        "namespace": "ep",
+        "type": "logs"
+    },
+    "ecs": {
+        "version": "8.4.0"
+    },
+    "elastic_agent": {
+        "id": "eb140869-9022-4dec-b977-6e198e7219b2",
+        "snapshot": true,
+        "version": "8.4.0"
+    },
+    "event": {
+        "agent_id_status": "verified",
+        "code": "5000000",
+        "dataset": "trendmicro.deep_security",
+        "ingested": "2022-10-30T04:54:23Z",
+        "severity": 5
```

```suggestion
        "severity": 5
        "category": "some_category",
         "type": "some_type"
```
If you can, please add a type & category attribute to the event object, since this are standard fields for our integrations.